src/domains/ioweb-common/01_network.tf

```diff
@@ -21,6 +21,12 @@ data "azurerm_subnet" "apim_v2_snet" {
   resource_group_name  = local.vnet_common_resource_group_name
 }
 
+data "azurerm_subnet" "ioweb_profile_snet" {
+  name                 = format("%s-ioweb-profile-snet", local.product)
```

Comment:
ditto

Reply:
fixed in https://github.com/pagopa/io-infra/pull/679/commits/68d692316214b8cd97fc28da3195bdf7f818de2c